libs/sdk-model/src/ldm/metadata/attributeDisplayForm/tests/index.test.ts

```diff
@@ -27,7 +27,9 @@ describe("attribute display form", () => {
 
     describe("attributeDisplayFormRef", () => {
         it("should return ref", () => {
-            expect(attributeDisplayFormRef(attributeDisplayForm)).toEqual(_attributeDisplayFormRef);
+            expect(attributeDisplayFormMetadataObjectRef(attributeDisplayForm)).toEqual(
```

Comment:
How is it possible this was passing / we did not discovered it sooner?

Reply:
test was not running because file name was without "test" suffix . And build process has not reference to it. I was surprised too :-)